### DISCUSSION TITLE
Icon from Papirus Icon Theme is not used at systray

### DISCUSSION THREAD

**Discussion by bupy7** [flameshot-org/flameshot#3530] [Q&A] [answered]
Hello!

```
Arch Linux
i3 version 4.23 (2023-10-29) © 2009 Michael Stapelberg and contributors

Flameshot v12.1.0 (-)
Compiled with Qt 5.15.10
```

I see predefined icon of Flameshot at my systray instead of Papirus Icon Theme. Other application like Telegram using Qt display icon from Papirus Icon Theme correctly. What should I check to fix it? I have icon `flameshot-tray` in my icons, but Flameshot doesn't use its:

```
Papirus/16x16/panel/flameshot-tray.svg
Papirus/24x24/panel/flameshot-tray.svg
Papirus/22x22/panel/flameshot-tray.svg
Papirus-Light/16x16/panel/flameshot-tray.svg
Papirus-Light/24x24/panel/flameshot-tray.svg
Papirus-Light/22x22/panel/flameshot-tray.svg
```

Thanks!

![image](https://github.com/flameshot-org/flameshot/assets/5145037/cc1fa2bd-191a-47d3-9975-6beb0e10a3d6)

**Comment by bupy7** [comment, marked as answer]
This may fix if call flameshot with `QT_QPA_PLATFORMTHEME=gtk3 flameshot`, but this is not a fix for `flameshot gui` call. It doesn't work with `gui` command. After that I have thought I don't need icon in systray at all anyway =) And just have hided it.

**Comment by mmahmoudian** [comment]
I think your fix would do, just you need to define that variable as a global variable. Perhaps putting it in your i3 config or `~/.xprofile`:

https://wiki.archlinux.org/title/Display_manager#Session_configuration

**Reply by bupy7** [reply]
That's right. Better place will be `.bash_profile` for me.
